frontend/src/app/admin/aima-data/page.tsx

```
@@ -725,60 +723,48 @@ export default function AdminQuestions() {
               <button
                 onClick={handleDownloadWaitlistEmails}
                 disabled={downloadingEmails}
-                className="inline-flex items-center px-6 py-3 bg-gradient-to-r from-green-600 to-green-700 hover:from-green-700 hover:to-green-800 disabled:from-gray-400 disabled:to-gray-500 text-white font-medium rounded-xl transition-all duration-200 transform hover:scale-105 shadow-lg hover:shadow-green-500/25 disabled:transform-none disabled:cursor-not-allowed"
+                className="inline-flex items-center px-6 py-3 bg-secondary hover:bg-secondary/80 disabled:opacity-50 text-secondary-foreground font-medium rounded-xl transition-all duration-200 transform hover:scale-105 shadow-sm disabled:transform-none disabled:cursor-not-allowed"
               >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)